### PR TITLE
Revise task page to deploy and access the Kubernetes Dashboard

### PR DESCRIPTION
**Pull request by sftim**
Update https://kubernetes.io/docs/tasks/access-application-cluster/web-ui-dashboard/ with a new title, a page description, and some rewording.

Old title: **Web UI (Dashboard)** (doesn't imply a task)
New title: **Deploy and Access the Kubernetes Dashboard**

/kind cleanup
/language en

### CODE REVIEW

File: content/en/docs/tasks/access-application-cluster/web-ui-dashboard.md [hunk 1]
```diff
@@ -203,23 +209,24 @@ this can be changed using the namespace selector located in the navigation menu.
 
 Dashboard shows most Kubernetes object kinds and groups them in a few menu categories.
 
-#### Admin Overview
-For cluster and namespace administrators, Dashboard lists Nodes, Namespaces and Persistent Volumes and has detail views for them.
+#### Admin overview
+
+For cluster and namespace administrators, Dashboard lists Nodes, Namespaces and PersistentVolumes and has detail views for them.
```
Comment (tylerauerbeck) [inline content/en/docs/tasks/access-application-cluster/web-ui-dashboard.md:214]: I think "and has detail views for each of them" may sound better here. This is just a small nit though. Feel free to disregard.

**Review by tylerauerbeck — COMMENTED** [review]
@sftim I have no review power here, but this LGTM. Just one small comment that you can feel free to disregard if you don't agree. Otherwise, hope this review can help to move things along!

**Comment by reylejano** [comment]
Preview of modified page:
https://deploy-preview-29515--kubernetes-io-main-staging.netlify.app/docs/tasks/access-application-cluster/web-ui-dashboard/

File: content/en/docs/tasks/access-application-cluster/web-ui-dashboard.md [hunk 2]
```diff
@@ -203,23 +209,24 @@ this can be changed using the namespace selector located in the navigation menu.
...
 
 #### Workloads
 
 Shows all applications running in the selected namespace.
-The view lists applications by workload kind (e.g., Deployments, Replica Sets, Stateful Sets, etc.)
+The view lists applications by workload kind (for example: Deployments, ReplicaSets, StatefulSets).
 and each workload kind can be viewed separately.
 The lists summarize actionable information about the workloads,
-such as the number of ready pods for a Replica Set or current memory usage for a Pod.
+such as the number of ready pods for a Replica et or current memory usage for a Pod.
```
Comment (reylejano) [inline content/en/docs/tasks/access-application-cluster/web-ui-dashboard.md:225]:
```suggestion
such as the number of ready pods for a ReplicaSet or current memory usage for a Pod.
```
typo

**Pull request merged**